### DISCUSSION TITLE
Custom Output Location/Folder

### DISCUSSION THREAD

**Discussion by CoffeeIsDead** [Monitor221hz/Pandora-Behaviour-Engine-Plus#139] [Ideas]
I'm not sure if it is possible, i.e. not sure what files are created/edited, but a custom output folder option would be nice for Vortex users so the can build once for their mod profiles and not have to launch every time they switch profiles. Loving the engine thus far, way faster and so much better then the juggle of using FNIS and Nemesis together when needed!

**Comment by Tyrthemis** [comment]
+1 to this as a vortex user, this was a big hang up in testing. Getting it to produce its own mod in a specified output location each time like DynDOLOD or TexGen would be perfect.  I had to do a clean reinstall to remove a massive amount of animations

**Comment by Monitor221hz** [comment]
@CoffeeIsDead @Tyrthem there is now a startup argument available for output folder in v1.3.0-alpha

**Reply by CoffeeIsDead** [reply]
Awesome!

**Reply by Tyrthemis** [reply]
Excellent, what is it? Is it the exact same as a tool like DynDOLOD? -o:[filepath]

**Reply by Monitor221hz** [reply]
> Excellent, what is it? Is it the exact same as a tool like DynDOLOD? -o:[filepath]

yes
